openedx/core/djangoapps/content/__init__.py

```diff
@@ -2,3 +2,7 @@
 Setup the signals on startup.
 """
 import openedx.core.djangoapps.content.course_structures.signals
+
+# Importing this in this manner seems to resolve some inexplicable circular import issue.
```

Comment:
How can this be reproduced?  Is it enough to remove the import here and run the tests to trigger the problem?